app/controllers/api/v0/signatures_controller.rb

```diff
@@ -1,21 +1,38 @@
 class Api::V0::SignaturesController < ApiController
   def create
     manifest = find_manifest(params[:manifest_id])
-    sign_request = read_body_as_json
+    signature_request = prep_signature_request(manifest)
 
-    manifest_content = manifest[:content].to_json
-    sign_request[:manifest_content] = manifest_content
-    system_session_token = session[:system_session_token]
-    sign_request["token"] = system_session_token
+    cdx_start = Time.now
```

Comment:
`Time.current` so we can ensure UTC time? http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/